spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'order'
+require 'menu'
+require 'takeaway'
+
+describe 'User Stories' do
+
+  describe 'Menu' do
+    # As a customer
+    # So that I can check if I want to order something
+    # I would like to see a menu§ of dishes with prices
+    it 'should display a list of dishes with prices' do
+      menu = Menu.new
+      expect(menu.list).not_to be_empty
+    end
+  end
+
+  describe 'Order' do
+    # As a customer
+    # So that I can order the meal I want
+    # I would like to be able to select some number of several available dishes
+    it 'should allow user to select some number of several dishes' do
+      order = Order.new(Menu.new.list)
+      order.choose("Dish 1", 2)
+      expect(order.dishes).not_to be_empty
+    end
+
+    # As a customer
+    # So that I can verify that my order is correct
+    # I would like to check that the total I have been given matches the sum of
+    # the various dishes in my order
+    it "the customer's total should match the sum of dishes in the order" do
+      takeaway = Takeaway.new(Order)
+      takeaway.create_order(Menu.new.list)
+      takeaway.order.choose("Dish 1", 3)
+      takeaway.order.calculate_quantities(takeaway.order.dishes)
+      takeaway.order.calculate_cost(takeaway.order.dishes)
+      expect{ takeaway.place_order(takeaway.order,takeaway.order.quantity,takeaway.order.total, ENV["TWILIO_SMS_NUMBER"]) }.not_to raise_error
```

Comment:
Metrics/LineLength: Line is too long. [142/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)